inference/aligner.py

```diff
@@ -258,7 +259,8 @@ def get_data(self, cv, z, bbox, src_mip, dst_mip, to_float=True,
       print('Normalizing image')
       start = time()
       data = torch.from_numpy(data)
-      data = data.to(device=self.device)
+      if gpu:
```

Comment:
Wouldn't it be cleaner to just set `self.device` to `'cpu'` when creating the aligner object whenever you want to run it on a cpu cluster?